configs/tao/ft_qdtrack_frcnn_r50_fpn_24e_tao.py

```diff
@@ -0,0 +1,253 @@
+# model settings
```

Comment:
Give these two config files are quite similar for most items.
I suggest let one inherit another. You can examples in mmdet